node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -737,6 +736,295 @@ mod tests {
         assert_eq!(config.crash_point, CrashPoint::Panic);
     }
 
+    fn fill_up_config_file(mut config_file: File) {
+        {
+            config_file
+                .write_all(b"real-user = \"1002:1002:/home/wooga\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"blockchain-service-url = \"https://www.mainnet2.com\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"clandestine-port = \"7788\"\n")
+                .unwrap();
+            config_file.write_all(b"consuming-private-key = \"00112233445566778899AABBCCDDEEFF00112233445566778899AABBCCDDEEFF\"\n").unwrap();
+            config_file.write_all(b"crash-point = \"None\"\n").unwrap();
+            config_file
+                .write_all(b"dns-servers = \"5.6.7.8\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"earning-wallet = \"0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\"\n")
+                .unwrap();
+            config_file.write_all(b"gas-price = \"77\"\n").unwrap();
+            config_file.write_all(b"log-level = \"trace\"\n").unwrap();
+            config_file
+                .write_all(b"mapping-protocol = \"pcp\"\n")
+                .unwrap();
+            config_file.write_all(b"min-hops = \"6\"\n").unwrap();
+            config_file
+                .write_all(b"neighborhood-mode = \"zero-hop\"\n")
+                .unwrap();
+            config_file.write_all(b"scans = \"off\"\n").unwrap();
+            config_file.write_all(b"rate-pack = \"2|2|2|2\"\n").unwrap();
+            config_file
+                .write_all(b"payment-thresholds = \"3333|55|33|646|999|999\"\n")
+                .unwrap();
+            config_file
+                .write_all(b"scan-intervals = \"111|100|99\"\n")
+                .unwrap()
+        }
+    }
+
+    #[test]
+    fn server_initializer_collected_params_handle_config_file_from_environment_and_real_user_from_config_file_with_path_started_by_dot() {
```

Comment:
"handleS"

You are mentioning the config file twice in the name, you can put the two parts together and get a simpler name.

Reply:
You will want the Environment guard here too because this tests refers to the environment setting.